extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -21,13 +21,16 @@ class SEOTweaksHooksHelper {
 	 * @return bool true
 	 */
 	static function onBeforePageDisplay( $out ) {
-		global $wgSEOGoogleSiteVerification, $wgSEOGooglePlusLink;
+		global $wgSEOGoogleSiteVerification, $wgSEOGooglePlusLink, $wgServer;
 		if ( !empty( $wgSEOGoogleSiteVerification ) ) {
 			$out->addMeta( 'google-site-verification', $wgSEOGoogleSiteVerification );
 		}
 		if ( !empty( $wgSEOGooglePlusLink ) ) {
 			$out->addLink( array( 'href' => $wgSEOGooglePlusLink, 'rel' => 'publisher' ) );
 		}
+		if ( Wikia::isMainPage() ) {
+			$out->addLink( [ 'href' => $wgServer, 'rel' => 'alternate', 'media' => 'only screen and (max-width: 640px)' ] );
```

Comment:
Maybe reorder these so that it outputs in the same order as the other links, e.g. ```rel```, ```media```, ```href```.  Currrently it looks like:
```html
<link href="http://garth.garth.wikia-dev.com" rel="alternate" media="only screen and (max-width: 640px)" />
<link rel="canonical" href="http://garth.garth.wikia-dev.com/wiki/Garth_Wiki" />
<link rel="alternate" type="application/x-wiki" title="Edit" href="/wiki/Garth_Wiki?action=edit" />
<!-- 7 more links starting w/ rel -->
```